socorro/cron/jobs/ftpscraper.py

```diff
@@ -25,6 +25,17 @@
 from socorro.lib.datetimeutil import string_to_datetime
 
 
+def memoize_download(fun):
+    cache = {}
+
+    def inner(self, url):
```

Comment:
My bad--this should have a `functools.wraps` around it. I forgot that in my earlier comment.